sdk/personalizer/Azure.AI.Personalizer/Azure.AI.Personalizer.sln

```diff
@@ -0,0 +1,39 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
We don't need this file, do we?
We have the solution file at the root folder of the service (sdk/personalizer/).